Framework/Geometry/src/Objects/Object.cpp

```diff
@@ -1644,7 +1644,23 @@ void Object::calcBoundingBoxByGeometry() {
       maxZ = std::max(maxZ, vector.Z());
     }
   } break;
-
+  case 2: // HEXAHEDRON
```

Comment:
There is a comment on the `default:` label below, which says that `2` is a SPHERE. Is the comment wrong, it the `2` here?